.github/workflows/lint-rust/action.yml

```diff
@@ -14,7 +14,9 @@ runs:
           with:
               submodules: recursive
 
-        - uses: dtolnay/rust-toolchain@stable
+        - uses: dtolnay/rust-toolchain@1.76.0
+          with:
+              components: "rustfmt, clippy"
```

Comment:
Had to add these components for some reason. CI was failing because these components weren't installed using the older toolchain version.